30. Lấy 1 comment(id_comment, comment) mới nhất và thông tin user của user đang được follow bởi user 1.md

```diff
@@ -0,0 +1,13 @@
+### 30. Lấy 1 comment(id_comment, comment) mới nhất và thông tin user của user đang được follow bởi user 1
+```mysql
+select comment.user_id, comment.comment, user.*
+from comment
+join user on comment.user_id  in (
```

Comment:
`comment` join `user` sao lại `comment.user_id  in ...`? anh có thể giải thích cho em chỗ này được không?

Reply:
join o day de lay thong tin user do